doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1747,52 +1702,57 @@ name of the variable (here :g:`n`) is chosen based on :g:`T`.
+   The second form refines an existential variable selected by its position.  The
    :n:`@natural` argument is the position of the existential variable
-   *from right to left* in the conclusion of the goal.  (Use one of
-   the variants below to select an existential variable in a
+   *from right to left* in the goal.  (Use the :n:`@hloc` clause
+   to select an existential variable in a
    hypothesis.)  Counting starts at 1 and multiple occurrences of the
-   same existential variable are counted multiple times.  Because this
-   variant is not robust to slight changes in the goal, its use is
-   strongly discouraged.
-
-.. tacv:: instantiate ( @natural := @term ) in @ident
-          instantiate ( @natural := @term ) in ( value of @ident )
-          instantiate ( @natural := @term ) in ( type of @ident )
-
-   These allow to refer respectively to existential variables occurring in a
-   hypothesis or in the body or the type of a local definition (named :n:`@ident`).
+   same existential variable are counted multiple times.  Using this form
+   is discouraged because slight changes to the goal may change the needed index,
+   causing a maintenance issue.
 
-.. tacv:: instantiate
+   Advanced users may want to define and use an Ltac tactic to get more consistent
+   behavior, such as:
 
-   This tactic behaves functionally as :tacn:`idtac`.
+   .. coqdoc::
 
-   .. deprecated:: 8.16
+      Ltac instantiate_ltac_variable ev term :=
+        let H := fresh in
+        pose ev as H;
+        instantiate (1 := term) in (value of H);
+        clear H.
 
-.. tacn:: admit
-   :name: admit
+   :n:`in @ident`
+     Selects the hypothesis :n:`@ident`.
 
-   This tactic allows temporarily skipping a subgoal so as to
-   progress further in the rest of the proof. A proof containing admitted
-   goals cannot be closed with :cmd:`Qed` but only with :cmd:`Admitted`.
+   :n:`in %|- *`
+     Selects the goal.  This is the default behavior.
 
-.. tacv:: give_up
+   :n:`in ( type of @ident )`
+     Selects existential variables in the type of the
+     :term:`local definition <context-local definition>` :n:`@ident`.
+     (The body is not included.)
 
-   Synonym of :tacn:`admit`.
+   :n:`in ( value of @ident )`
+     Selects existential variables in the body of the
+     :term:`local definition <context-local definition>` :n:`@ident`.
+     (The type is not included.)
 
-.. tacn:: absurd @term
-   :name: absurd
+.. tacn:: absurd @one_type
 
-   This tactic applies to any goal. The argument term is any proposition
+   :n:`@one_type` is any proposition
    :g:`P` of type :g:`Prop`. This tactic applies False elimination, that is it
    deduces the current goal from False, and generates as subgoals :g:`∼P` and
    :g:`P`. It is very useful in proofs by cases, where some cases are
    impossible. In most cases, :g:`P` or :g:`∼P` is one of the hypotheses of the
    local context.
 
-.. tacn:: contradiction
+.. tacn:: contradiction {? @one_term_with_bindings }
```

Comment:
Leave out the grammar fix until we update the description and merge it with the one of its variant.

```suggestion
.. tacn:: contradiction
```

Reply:
Note that `doc_grammar` updates the `:tacn:` automatically.  I have to tweak `common.edit_mlg` to avoid that.